<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Common2ParamWidget</class>
 <widget class="QWidget" name="Common2ParamWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>673</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>673</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QWidget" name="pop_widget_title" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777214</width>
       <height>35</height>
      </size>
     </property>
     <widget class="QPushButton" name="pop_cloase_btn">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>6</y>
        <width>24</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pop_min_btn">
      <property name="geometry">
       <rect>
        <x>535</x>
        <y>6</y>
        <width>24</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="title_lab">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>3</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
color: rgb(0, 200, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QPushButton" name="pop_cancle_btn">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>540</y>
        <width>95</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pop_commit_btn">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>540</y>
        <width>95</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="tagName_lab">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(117, 117, 117);
font: 75 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="tag_name_Edit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="remake_Edit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>100</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="remake_lab">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(117, 117, 117);
font: 75 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>备注：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
